### DISCUSSION TITLE
Vision plugin crashes my next app

### DISCUSSION THREAD

**Discussion by lourencelinao** [sanity-io/sanity#3772] [Sanity Studio Support] [answered]
My next app crashes when I try to add the vision plugin.

![image](https://user-images.githubusercontent.com/39908576/196581153-46e6d90c-e314-47cd-8841-ad5bbfe0ddee.png)

Here is the package version.
```
"@sanity/vision": "3.0.0-purple-unicorn-starter.43",
"sanity": "3.0.0-purple-unicorn-starter.43"
```

**Comment by lobunto** [comment, marked as answer]
@lourencelinao Have you put `visionTool()` inside the `plugins` property? Like this:

```js
export default createConfig({
  name: 'projectName'.
  basePath: '/studio',
  plugins: [deskTool({ structure }), visionTool({ })]
})
```

**Reply by lourencelinao** [reply]
Hello @lobunto, thanks for responding, yes i put it inside the plugins property. It wasn't working earlier but now it is working and I didn't do anything to fix it 🤷. I just uncommented the visionTool code and it started working.

**Reply by marcobarbosa** [reply]
I got the exact same error. @lourencelinao Is it still working for you?

These are my versions though:

```
"sanity": "^3.0.0-dev-preview.20",
"@sanity/vision": "^3.0.0-dev-preview.20",
```

**Reply by lourencelinao** [reply]
> I got the exact same error. @lourencelinao Is it still working for you?
> 
> These are my versions though:
> 
> ```
> "sanity": "^3.0.0-dev-preview.20",
> "@sanity/vision": "^3.0.0-dev-preview.20",
> ```

Yes it is working for me. 

I was using the same version as yours before, what worked for me is I changed to the package version that I am currently using.

**Reply by marcobarbosa** [reply]
Thanks for the reply, @lourencelinao .

And what is this "3.0.0-purple-unicorn-starter.43"? Where did you find this version? 

I feel like I have to hunt for information since I'm following what's described in the official docs :(

**Reply by marcobarbosa** [reply]
I just read that preview.22 got released and now it works. Ignore my ranting :) 

```
    "sanity": "^3.0.0-dev-preview.22",
    "@sanity/vision": "^3.0.0-dev-preview.22",
```
